Scripts/RemediationScripts/Remediate-ConfigureConditionalAccessPolicyForPIM.ps1

```diff
@@ -311,7 +303,7 @@ function Configure-ConditionalAccessPolicyForPIM
             $roleId = $policyId.PolicyId.Split('/')
         
             $policyDetails = Get-AzRoleManagementPolicy -Scope "subscriptions/$($SubscriptionId)" -Name $roleId[-1]
-            $configuredPolicyDetail = $policyDetails.EffectiveRule | Where-Object {$_.claimValue -eq "urn:microsoft:req1" -and $_.IsEnabled -eq $true}
+            $configuredPolicyDetail = $policyDetails.EffectiveRule | Where-Object {$_.claimValue -eq "c1" -and $_.IsEnabled -eq $true}
```

Comment:
What if user has previously set old value for few roles. shall we keep OR condition here as well.

Reply:
Yes, added condition to handle old values